dts/bindings/spi/nordic,nrf-spi.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: BSD-5-Clause-Nordic
+#
+---
+title: Nordic nRF Family SPI Master node
+id: nordic,nrf-spi
+version: 0.1
+
+description: >
+    This is a representation of the Nordic nRF SPI node
+
+inherits:
+    !include spi.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nordic,nrf-spi"
+
+    reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+    interrupts:
+      type: array
+      category: required
+      description: required interrupts
+      generation: define
+
+    sclk-pin:
```

Comment:
In all our Product Specifications this pin is referred as the "SCK pin". I think we should stick to this convention.